### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by Stephen-Adom**
# Vet clinic database: add "join table" for visits
In this pull request, the following tasks were implemented:
- Created a table named vets with the following columns:
    - id: integer (set it as autoincremented PRIMARY KEY)
    - name: string
    - age: integer
    - date_of_graduation: date

- Inserted the following data for vets:
    - Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
    - Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
    - Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
    - Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.

- Inserted the following data for specialties:
    - Vet William Tatcher is specialized in Pokemon.
    - Vet Stephanie Mendez is specialized in Digimon and Pokemon.
    - Vet Jack Harkness is specialized in Digimon.

- Inserted data into visit table.
- Below is the snapshot of query results:

Screenshot from 2023-05-04 13-17-11

Screenshot from 2023-05-04 13-17-33

Screenshot from 2023-05-04 13-17-57

Screenshot from 2023-05-04 13-18-27

Screenshot from 2023-05-04 13-18-52

Screenshot from 2023-05-04 13-19-14

Screenshot from 2023-05-04 13-19-40

Screenshot from 2023-05-04 13-20-01

Screenshot from 2023-05-04 13-20-24

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -61,3 +61,50 @@ UPDATE animal SET owner_id = 4 WHERE name = 'Blossom';
...
 UPDATE animal SET owner_id = 5 WHERE name = 'Boarmon';
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('William Tatcher', 45, date '2000-04-23');
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('Maisy Smith', 26, date '2019-01-17');
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('Stephanie Mendez', 64, date '1981-05-04');
+
+INSERT INTO vet(name, age, date_of_birth) VALUES('Jack Harkness', 38, date '2008-06-08');
+
```
Comment (GedeonTS) [inline data.sql:72]: - Good Job implementing the requirements to your project. However, according to [this requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md#:~:text=Insert%20the%20following%20data%20for%20specialties,Jack%20Harkness%20is%20specialized%20in%20Digimon.), your are supposed to insert data into the Specialization join table. Kindly add the insertion data into this data.sql file

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @Stephen-Adom , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good PR Description :+1: 
- Good Summary :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Stephen-Adom) [reply data.sql:72]: @GedeonTS , I forgot to add the SQL queries to the data.sql file. Thank you for letting me know. I will do that immediately.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Stephen-Adom 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes from the previous reviewer 

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**